### DISCUSSION TITLE
from_dttm not working anymore

### DISCUSSION THREAD

**Discussion by noxonnor** [apache/superset#19619] [Q&A / Help]
Hi,

We upgraded to 1.4.2 recently, and we noticed some of our reports no longer works. Specifically, we use the {{from_dttm}} and {{to_dttm}} filter to get the time range for filtering a query. 

Error message:
Failed to execute query '126' - 'SELECT
        {{from_dttm}} end_dt
           FROM DUAL': The parameter "from_dttm" in your query is undefined. 
 Please check your template parameters for syntax errors and make sure they match across your SQL query and Set Parameters. Then, try running your query again.

Whereas doing the same with a parameter such as {{current_username()}}  works without any issue..

We've turned on 
DEFAULT_FEATURE_FLAGS = {
    "ENABLE_TEMPLATE_PROCESSING": True
}
in the config. 

Any ideas?

**Comment by ss2495** [comment]
I am also facing the same issue in superset.

**Comment by spacemudd** [comment]
You have to add the parameters.

https://youtu.be/CdZy3LivkhQ?t=1287
